src/styles/cards-container.scss

```diff
@@ -0,0 +1,21 @@
+.cards-container {
+  justify-content: center;
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($card-width, 1fr));
+  gap: 46px 48px;
+  max-width: $container-width;
```

Comment:
Ensure that the `$container-width` variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.